### PR TITLE
Fix remove news and add news bug

### PR DESCRIPTION
**Pull request by milana986**

### CODE REVIEW

File: src/main/webapp/static/app/components/news/news.html [hunk 1]
```diff
@@ -14,27 +12,30 @@
...
-      <span class="pull-right">{{vest.created}}</span>
-      <!--<p>{{vest.description}}</p>-->
-      <p>{{vest.text}}</p>
-      <button class="btn btn-danger pull-right" ng-click="removeNews(vest)">Obrisi ovu vest</button>
-    </div>
-    <!--</div>-->
+<hr>
+
+<div class="col-sm-12 mainContent">
+  <!--<div ng-class="{'row': $index % 3 == 2}">-->
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/news.html]: Please remove this is not use.

File: src/main/webapp/static/app/components/news/news.html [hunk 2]
```diff
@@ -14,27 +12,30 @@
...
+  <div class="col-md-4 col-sm-6 grey" ng-repeat="vest in news">
+    <h3>{{vest.title}} </h3>
+    <br>
+    <span class="pull-right">{{vest.created}}</span>
+    <!--<p>{{vest.description}}</p>-->
+    <p>{{vest.text}}</p>
+    <button class="btn btn-success pull-right" ui-sref="main.editNews({id:vest.id})">Edit</button>
+    <button class=" btn btn-danger pull-right" ng-click="removeNews(vest)">Obrisi ovu vest</button>
   </div>
+  <!--</div>-->
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/news.html]: also

File: src/main/webapp/static/app/components/news/news.service.js [hunk 3]
```diff
@@ -18,21 +18,30 @@
       });
     };
 
-    var addNews = function(oneNews) {  //return $http.post(url + "/create", oneNews);  //moze i samo tako da se zavrsi
+    var saveEditNews = function(oneNews) {  //return $http.post(url + "/create", oneNews);  //moze i samo tako da se zavrsi
       return $http.post(url + "/create", oneNews).then(function(response) {
         console.log(response.data);
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/news.service.js]: remove console.log

File: src/main/webapp/static/app/components/news/news.service.js [hunk 4]
```diff
@@ -18,21 +18,30 @@
...
         return response.data;
       });
     };
 
+    var getById = function(id) {
+      return $http.get(url + "/" + id).then(function(response) {  //mozda zatreba params za search i filter
+        console.log(response.data);
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/news.service.js]: remove this also

File: src/main/webapp/static/app/components/news/news.service.js [hunk 5]
```diff
@@ -18,21 +18,30 @@
...
+
     var deleteNews = function(vest) {
       return $http.delete(url + "/remove/" + vest.id);
     };
 
     return {
       getAllnews: getAllnews,
-      addNews: addNews,
-      deleteNews: deleteNews
+      saveEditNews: saveEditNews,
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/news.service.js]: maybe should be called something like createAndEditNews

File: src/main/webapp/static/assets/css/style.css [hunk 6]
```diff
@@ -1,38 +1,45 @@
 
-.bkg{
-	background: #320f77;
-/*	position: fixed;*/
+.bkg {
+  background: #320f77;
+  /*	position: fixed;*/
```
Comment (DraganU) [inline src/main/webapp/static/assets/css/style.css]: what is this?

File: src/main/webapp/static/assets/css/style.css [hunk 7]
```diff
@@ -1,38 +1,45 @@
...
 }
 
-.sidebar{
-	border: 1px solid blue;
-	height: 30em;
-	
+.sidebar {
+  border: 1px solid blue;
+  height: 30em;
+
```
Comment (DraganU) [inline src/main/webapp/static/assets/css/style.css:27]: remove this empty line please.

File: src/main/webapp/static/app/components/news/newsDetail.html [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
       <option>STANDARD</option>
     </select>
   </div>
   <div class="form-group">
     <label for="sel">Category :</label>
-    <select class="form-control" id="sel" ng-model="newNews.category">
+    <select class="form-control" id="sel" ng-model="news.category">
       <option>POLITIKA</option>
       <option>EKONOMIJA</option>
       <option>KULTURA</option>
```
Comment (DraganU) [inline src/main/webapp/static/app/components/news/newsDetail.html]: This categories extract to one array in global varible inside scope.

File: src/main/webapp/static/app/components/news/news.html (hunk 2 above)
Comment (milana986) [reply src/main/webapp/static/app/components/news/news.html]: moved

File: src/main/webapp/static/app/components/news/news.service.js (hunk 3 above)
Comment (milana986) [reply src/main/webapp/static/app/components/news/news.service.js]: done

File: src/main/webapp/static/app/components/news/news.service.js (hunk 4 above)
Comment (milana986) [reply src/main/webapp/static/app/components/news/news.service.js]: done

File: src/main/webapp/static/app/components/news/news.service.js (hunk 5 above)
Comment (milana986) [reply src/main/webapp/static/app/components/news/news.service.js]: createNewsAndEditNews? is it ok?

File: src/main/webapp/static/assets/css/style.css (hunk 6 above)
Comment (milana986) [reply src/main/webapp/static/assets/css/style.css]: removed

File: src/main/webapp/static/app/components/news/newsDetail.html (hunk 8 above)
Comment (milana986) [reply src/main/webapp/static/app/components/news/newsDetail.html]: done

**Pull request merged**